UPDATING.md

```diff
@@ -130,6 +130,62 @@ session.add(admin)
 session.commit()
 ```
 
+
+## Changes in Google Cloud Platform related operators
+
+Most GCP-related operators have now optional PROJECT_ID parameter. In case you do not specify PROJECT_ID,
+the PROJECT_ID configured in used GCP Connection is used. There will be an exception thrown in case
+neither is specified. This change should be backwards compatible as earlier version of the
+operators had PROJECT_ID mandatory.
```

Comment:
let's also make it explicit that "Other GCP operators not listed here are unaffected by this change."

Reply:
Done.

Reply:
It's not rare that people write DAGs directly with these GCP hooks, could you please also document this hook change?

Reply:
I did actually below. PTAL if that's what you mean. I even listed Hooks affected/

Reply:
@fenglu-g I've added extra header for the "Hooks" chapter - it was under "operators" chapter so that could be misleading.